lib/SlackBot.js

```diff
@@ -2,6 +2,7 @@ var Botkit = require(__dirname + '/CoreBot.js');
 var request = require('request');
 var express = require('express');
 var bodyParser = require('body-parser');
+var querystring = require('querystring');
```

Comment:
if you're using https://www.npmjs.com/package/querystring querystring npm module, you should install the package with `npm install querystring --save` to save the node module name to package.json. Additionally, if you don't do it, and next time a developer installs the app using `npm install` the querystring node module will not be installed and therefore result in a crash.